packages/hash/frontend/src/theme/palette.ts

```diff
@@ -149,10 +149,45 @@ export const customColors = {
 } as const;
 
 export const palette: ThemeOptions["palette"] = {
-  ...customColors,
+  divider: customColors.gray[30],
+  primary: {
+    dark: customColors.blue[90],
+    main: customColors.blue[70],
+    light: customColors.blue[50],
+  },
+  secondary: {
+    dark: customColors.purple[90],
+    main: customColors.purple[70],
+    light: customColors.purple[50],
+  },
+  warning: {
+    dark: customColors.blue[80],
+    main: customColors.blue[60],
+    light: customColors.blue[40],
```

Comment:
Shouldn't warnings be orange?
```suggestion
    dark: customColors.orange[80],
    main: customColors.orange[60],
    light: customColors.orange[40],
```

Reply:
Not sure how I missed this. Thanks for catching! Fixed here => 439e6aa